### PR TITLE
del: add page

### PR DESCRIPTION
**Pull request by pxgamer**
Part of #1579

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/windows/del.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+- Delete one or more space-separated files or patterns:
+
+`del {{file_pattern}}`
+
+- Prompt for confirmation before deleting each file:
+
+`del {{file_pattern}} /p`
+
+- Force the deletion of read-only files:
```
Comment (agnivade) [inline pages/windows/del.md]: Why read-only ? Does it mean it won't delete writable files ?

File: pages/windows/del.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+`del {{file_pattern}} /f`
+
+- Recursively delete files from all subdirectories:
+
+`del {{file_pattern}} /s`
+
+- Do not prompt when deleting files based on a global wildcard:
```
Comment (agnivade) [inline pages/windows/del.md]: What does "based on a global wildcard" mean ?

File: pages/windows/del.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+- Prompt for confirmation before deleting each file:
+
+`del {{file_pattern}} /p`
+
+- Force the deletion of read-only files:
+
+`del {{file_pattern}} /f`
+
+- Recursively delete files from all subdirectories:
```
Comment (agnivade) [inline pages/windows/del.md]: Perhaps change this to `file(s)` ?

File: pages/windows/del.md (hunk 1 above)
Comment (pxgamer) [reply pages/windows/del.md]: By default, `del` only removes files that aren't set to `Read-only` on the file system (there's a checkbox in the file explorer).

Using `/f` will remove any files that have the `Read-only` status set, as well as other files.

File: pages/windows/del.md (hunk 3 above)
Comment (pxgamer) [reply pages/windows/del.md]: Sounds good 👍

File: pages/windows/del.md (hunk 2 above)
Comment (pxgamer) [reply pages/windows/del.md]: An example would be `del *` or `del *.*` (both do the same thing).

File: pages/windows/del.md (hunk 1 above)
Comment (agnivade) [reply pages/windows/del.md]: I see. Got it.

File: pages/windows/del.md (hunk 2 above)
Comment (agnivade) [reply pages/windows/del.md]: But that has nothing to do with the `/q` switch right ? You can do `del file.txt /q` and `del *.txt /q` and both will do the same thing. The description makes it look like the the opposite is also true - i.e. when files are deleted without a wildcard, it prompts even if you specify `/q`.

Comment (pxgamer) [reply pages/windows/del.md]: The `/q` switch is only used to allow deletion of everything in a directory using either `*` or `*.*`. Specifying a wildcard with an extension (e.g. `*.txt`) won't prompt. You're right that `/q` won't do anything for `*.txt` or `file.txt` as it only applies when a wildcard is used that will delete all content.

Comment (agnivade) [reply pages/windows/del.md]: Ah I see now. Perhaps you should just specify * or *.* in brackets. Because it applies only on those cases.

Comment (pxgamer) [reply pages/windows/del.md]: Sounds good. :+1: Would improve the readability and reduce confusion.

File: pages/windows/del.md (hunk 1 above)
Comment (pxgamer) [reply pages/windows/del.md]: Ping @agnivade. Is this ok? 🤔

**Comment by sbrl** [comment]
Merged due to inactivity and the PR being made by a maintainer.

**Pull request merged**